docs/storage/commit_log/simkafka/kafka_test.go

```diff
@@ -54,7 +54,7 @@ func TestRead(t *testing.T) {
 	}
 	offset = 2
 	if got := Read(topic, offset); got != "" {
-		t.Errorf("Read(%q, %d)=%v; want ''", topic, got, offset)
+		t.Errorf("Read(%q, %d)=%v; want ''", topic, offset, got)
```

Comment:
Good spot!  Looks like 3 lines down has the same problem -- do you want to fix that too while you're there?